pkg/userdata/centos/testdata/kubelet-v1.12-vsphere.golden

```diff
@@ -59,6 +59,11 @@ write_files:
     systemctl restart systemd-modules-load.service
     sysctl --system
 
+    # Make sure we always disable swap - Otherwise the kubelet won't start
+    mv /etc/fstab /etc/fstab.orig
+    cat /etc/fstab.orig | awk '$3 ~ /^swap$/ && $1 !~ /^#/ {$0="# commented out by cloudinit\n#"$0} 1' > /etc/fstab
```

Comment:
Can we please create a /etc/fstab.noswap first and then mv it atomically onto /etc/fstab?
If anything breaks after your first mv you'll have a system without any /etc/fstab

Reply:
Yes, I like that idea. Let's do that. Update coming in 5 mins...

Reply:
Updated.